### PR TITLE
libpll-1.0.11

### PR DESCRIPTION
**Pull request by kgori**
### Have you:

- [ ] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [ ] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [ ] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

**Comment by iMichka** [comment]
@BrewTestBot test this please

File: libpll.rb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
   bottle do
-    root_url "https://homebrew.bintray.com/bottles-science"
     cellar :any
     sha256 "6d4886d7286af7ba1c7f4615bd1aab9af0a87a72d72767d8659b836649059a10" => :yosemite
     sha256 "0f9f835ab42cace506d68c8e90e47d5a558a0387297672592a1736057666c6bd" => :mavericks
     sha256 "7b0a93079fdbb1f063c940cb054dbba66267c06e6238fa40b5d96bbe67d77201" => :mountain_lion
   end
 
+  option "without-test", "Disable build-time checking (not recommended)"
```
Comment (jonchang) [inline libpll.rb]: This needs to use `deprecated_option` so users updating won't get errors.

```
Migrate '--without-check' with `deprecated_option`.
```

Comment (kgori) [inline libpll.rb]: Will adding this single line be enough?

    deprecated_option "without-check" => "without-test"

Comment (jonchang) [inline libpll.rb]: Yes, that should be fine. Rerun `brew audit --strict --online` to check that the audits pass and this should be good to go.

**Comment by jonchang** [comment]
Ok, this looks good. Will merge when the Linux bottles finish uploading.

**Comment by jonchang** [comment]
Thanks for your patience and hard work @kgori! This has been merged.

**Pull request closed without merging**